src/EmbeddedSonarAnalyzer.props

```diff
@@ -5,10 +5,10 @@
   <PropertyGroup>
     <!-- Note: Guide on how to update the analyzers is on the xtranet! -->
     <EmbeddedSonarAnalyzerVersion>8.53.0.62665</EmbeddedSonarAnalyzerVersion>
-    <EmbeddedSonarCFamilyAnalyzerVersion>6.42.0.61084</EmbeddedSonarCFamilyAnalyzerVersion>
+    <EmbeddedSonarCFamilyAnalyzerVersion>6.43.0.61486</EmbeddedSonarCFamilyAnalyzerVersion>
     <EmbeddedSonarJSAnalyzerVersion>10.0.1.20755</EmbeddedSonarJSAnalyzerVersion>
-	
-	<!-- Secrets
+
+    <!-- Secrets
```

Comment:
The release notes say "bug fixes and improvements" -> the number of rules and set of active rules hasn't changed, so there the test that checks the rules doesn't need to change.

I'll fix the extra whitespace before the `<!-- Secrets`.